src/lwip/BUILD.gn

```diff
@@ -225,6 +225,12 @@ if (current_os == "zephyr" || current_os == "mbed") {
           [ "${chip_root}/third_party/lwip/repo/lwip/src/apps/mdns/mdns.c" ]
     } else if (lwip_platform == "silabs") {
       public_deps += [ "${efr32_sdk_build_root}:efr32_sdk" ]
+
+      sources += [
+        "${lwip_platform}/lwipopts-rs911x.h",
+        "${lwip_platform}/lwipopts-thread.h",
```

Comment:
this one shouldn't be needed anymore

Reply:
ok, I will remove it.

Reply:
Although it is explicitly referenced in lwipopts.h ...

Reply:
it is referenced in lwipopts.h when none of the wifi boards are in use.

Reply:
@andy31415 The header or its include has no reason to exist anymore. we can delete all this. I can do it in a separate PR if you want. Lwip is not used for our thread platforms